mmi/runner.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 """
 Usage:
-  mmi-runner <engine> <configfile> [-o <outputvar>...] [-g <globalvar>...] [--interval <interval>] [--disable-logger] [--pause] [--mpi <method>] [--track <server>] [--port <port>]
+  mmi-runner <engine> <configfile> [-o <outputvar>...] [-g <globalvar>...] [--interval <interval>] [--disable-logger] [--pause] [--mpi <method>] [--track <server>] [--port <port>] [--bmi-class]
```

Comment:
If you look at the [docopt documation](http://docopt.org) you see that if an option has an argument (like `--bmi-class wflow.BMIModel` it should be specified using the `<>` or fullcaps notation. So the syntax should probably be `[--bmi-class <classname>]` instead of just `[--bmi-class]`.